### DISCUSSION TITLE
Welcome to XER Tools (Public) Discussions!

### DISCUSSION THREAD

**Discussion by DGH2112** [DGH2112/XER-Tools-Public-#3] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with the users of XER Tools. I hope that you can:
  * Ask questions you’re wondering about;
  * Share ideas, suggestion, features requests;
  * Engage with other users;
  * Where I can ask the community of users for feedback on suggested changes.

**Comment by DGH2112** [comment]
### 64-Bit XER Tools going forward?
I don't know how many of you use the command line tool to clean files but I was think for the next release (2.2, in the near future) to only release the 64-bit version of XER Tools as very few people in the world today run 32-bit windows and I'm not aware of anyone using the command line tool these days.
regards
Dave.

**Reply by fuguboy** [reply]
well, when they need to, they will use it. (I meant command prompt)
Furthermore, most OS are now 64bit as we can see the trending of supplied laptops.

**Comment by khaledra90** [comment]
Greetings David,
I really want to thank you for the effort that has been done for creating such an amazing tool.
I was wondering if you can add a new feature to XER Tools which is displaying activities by activity codes same as activities by WBS
also, can you add page same as Log file in P6 especially warning part.

Thanks in advance

**Comment by DGH2112** [comment]
Hi, sorry for not responding before now, work has been a little hectic.

Thanks for the appreciation.

- Regarding your first request, grouping by activities code, someone else has recently asked the same and I'll look to try and include that in the next release (2.3) however I have no idea how I will implement it in a performant manner at the moment.
- On point 2, I think you are asking for the scheduling log or similar from P6? At the moment I don't perform a full reschedule of the data, just enough to understand driving logic. The Schedule Integrity Checks would or should give you the issues with your programme or is there something missing from those checks you see in the log you want exposed?
- 
regards
Dave.

**Reply by khaledra90** [reply]
Thanks Dave for your response,
Hopefully you'll succeed to implement.
regarding the other point, the integrity report is very powerful and yes you are correct it's very sufficient.
one more thing, I was wondering if you able to simulate the filters same as P6 if applicable, at least when we click on the filter the list of values in the specific column to be visible same as excel.

B.R,
Khaled.

**Reply by DGH2112** [reply]
I'm reluctant to change the filters as they are because, for me, they are more powerful than Excel's quick filtering, however, I may be able to add functionality to the search criteria to allow you to select from a list and build a regular expression for you but it might be slow to build the list if you are working with large XER files (which we are).

**Reply by khaledra90** [reply]
Off course the most important thing is the performance specially as you mentioned when we are working with large schedule however you can add as an option can be enabled/disabled from options.

**Reply by DGH2112** [reply]
I was thinking that the edit control, which takes the filter criteria, could have an ellipsis button at the end, which would bring up a list of field values which can be filtered if required.
